src/how-to/render/display-text.md

```diff
@@ -1,5 +1,164 @@
 # Displaying Text
 
-(Stub) This page covers how text displaying works. It will cover `Line`, `Span`, `Text`, the
-migration of `Spans` to `Text`, and how these can be created, styled, displayed, altered, and such.
-These are all interrelated topics so, they are on the same page.
+This page covers how text displaying works. It will cover `Span`, `Line`, and `Text`, and how these
+can be created, styled, displayed, altered, and such.
+
+## `Span`
+
+A `Span` is a styled segment of text. You can think of it as a substring with its own unique style.
+It is the most basic unit of displaying text in `ratatui`.
+
+The examples below assume the following imports:
+
+```rust
+use ratatui::{prelude::*, widgets::*};
+pub type Frame<'a> = ratatui::Frame<'a, CrosstermBackend<std::io::Stderr>>;
+```
+
+A `Span` consists of "content" and a "style" for the content. And a `Span` can be created in a few
+different ways.
+
+1. using `Span::raw`:
+
+```rust
+fn ui(_app: &App, f: &mut Frame<'_>) {
+    let span = Span::raw("This is text that is not styled");
+    // -- snip ---
+}
+```
+
+2. using `Span::styled`:
+
+```rust
+fn ui(_app: &App, f: &mut Frame<'_>) {
+    let span = Span::styled("This is text that will be yellow", Style::default().fg(Color::Yellow));
+    // -- snip ---
+}
+```
+
+3. using the `Stylize` trait:
+
+```rust
+fn ui(_app: &App, f: &mut Frame<'_>) {
+    let span = "This is text that will be yellow".yellow();
+    // -- snip ---
+}
+```
+
+`Span`s are the building blocks to build styled content that is text that is represented
+horizontally.
+
+## `Line`
+
+The next building block that we are going to talk about is a `Line`. A `Line` represents a single
+"line" of text. The `Line` struct is essentially a `Vec<Span>`.
+
+Each Line comprises of multiple `Span` objects which allow for varied styling within a single line.
+
+```rust
+fn ui(_: &App, f: &mut Frame<'_>) {
+    let line = Line::from(vec![
+        "hello".red(),
+        " ".into(),
+        "world".red().bold()
+    ]);
+    // -- snip ---
+}
+```
+
+A `Line` can be constructed directly from content too.
```

Comment:
What is "content" in this context? Is it any string like object? I think this is fine, but also we might want to make it more clear so they don't have to go look up the "technical" explanation for what Line::from can take.

Reply:
You caught me trying to side step what `content` is. I didn't want to get into it because `Span`, `Line` and `Text` takes anything that is `Into<Cow<&str>>`. 

Maybe I should just say "where content is anything that can be converted to a copy on write string" and leave it at that?

Reply:
I think that would be fine. It lets the reader know there is more under the hood going on, and that there is more information available in the docs, but also doesn't leave them wondering what `content` is.

Reply:
Fixed in https://github.com/ratatui-org/ratatui-book/pull/72/commits/91e64b3162aba891f361c13105eb992c8fad86dc.